### PR TITLE
add java implementation to phidias

### PR DESCRIPTION
**Pull request by TruaShamu**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [x] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [x] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

### CODE REVIEW

**Comment by TruaShamu** [comment]
Proof that it works
phidias1
phidias2

**Review by jessechoe10 — APPROVED** [review]

File: solutions/dmoj-Phidias.mdx [hunk 1]
```diff
@@ -79,5 +79,66 @@ int main(){
 	cout << dp[w][h] << endl;
 }
 ```
-</CPPSection>				
+</CPPSection>
+
+<JavaSection>
+
+```java
+
```
Comment (SansPapyrus683) [inline solutions/dmoj-Phidias.mdx:87]:
```suggestion
```

File: solutions/dmoj-Phidias.mdx [hunk 2]
```diff
@@ -79,5 +79,66 @@ int main(){
...
+				}
+			}
+		}
+
+		// Answer.
+		System.out.println(dp[W][H]);
+	}
+
+	// CodeSnip{Kattio}
+
```
Comment (SansPapyrus683) [inline solutions/dmoj-Phidias.mdx:138]:
```suggestion
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
too lazy to commit these changees myself

**Review by SansPapyrus683 — APPROVED** [review]
enough comments to excuse the bad variable names

**Pull request merged**